pkgs/applications/misc/vym/default.nix

```diff
@@ -61,8 +65,8 @@ stdenv.mkDerivation (finalAttrs: {
       drawn by hand or any drawing software vym offers much more features to
       work with such maps.
     '';
-    license = licenses.gpl2Plus;
-    maintainers = [ maintainers.AndersonTorres ];
-    platforms = platforms.linux;
+    license = with lib.licenses; [ gpl2Plus ];
```

Comment:
Hi @AndersonTorres could you fill me in either with a link or comment why you're using `with` this way now?

Reply:
to be more clear, why it was move from `meta`

Reply:
My arguments (in the sense of "I agree with this", not in the sense of "I wrote this") against `with` are summarized here: https://nix.dev/guides/best-practices.html#with-scopes

Because of it I prefer to reduce the abuses of `with`, especially the `with` inside `with`.
A healthy compromise to me is restrict `with` to single-line statements.